pkg/lib/pb/model/protos/models.proto

```diff
@@ -1157,21 +1157,7 @@ message ManifestInfo {
     string language = 12;
 }
 
-message Membership {
-    enum Tier {
-        TierNewUser = 0;
-        // "free" tier
-        TierExplorer = 1;
-        // this tier can be used just for testing in debug mode
-        // it will still create an active subscription, but with NO features
-        TierBuilder1WeekTEST = 2;
-        // this tier can be used just for testing in debug mode
-        // it will still create an active subscription, but with NO features
-        TierCoCreator1WeekTEST = 3;
-        TierBuilder = 4;
-        TierCoCreator = 5;
-    }
-    
+message Membership {    
```

Comment:
Where are all these tiers identifiers are going? To any-sync protocol?

Reply:
Yes, If you want/need -> I can keep it here also and roll this change back

Reply:
No, it is better to keep it at any-sync, you are right